dag/main.yml

```diff
@@ -117,12 +113,26 @@ steps:
 
   #
   # UN
-  #
+  data://meadow/un_sdg/2022-07-07/un_sdg:
+    - walden://un_sdg/2022-07-07/un_sdg
+  data://garden/un_sdg/2022-07-07/un_sdg:
+    - data://meadow/un_sdg/2022-07-07/un_sdg
   data://grapher/un_sdg/2022-07-07/un_sdg:
     - data://garden/un_sdg/2022-07-07/un_sdg
   data://grapher/un/2022-07-11/un_wpp:
     - data://garden/un/2022-07-11/un_wpp
 
+ # UN SDG - current
+  data://meadow/un/2023-01-24/un_sdg:
```

Comment:
I've noticed we're changing namespace from `un_sdg` to `un`. Could you also change namespace of the old dataset from `un_sdg` to `un`? This would let us archive it. If we're lucky we won't have to make any changes to production DB or anything.